canvas_sdk/commands/tests.py

```diff
@@ -628,3 +626,83 @@ def test_command_allows_kwarg_with_correct_type(
     updated_k, updated_v = list(test_updated_value.items())[-1]
     setattr(cmd, updated_k, updated_v)
     assert getattr(cmd, updated_k) == updated_v
+
+
+@pytest.fixture
+def token() -> str:
+    return requests.post(
+        f"{settings.INTEGRATION_TEST_URL}/auth/token/",
+        headers={"Content-Type": "application/x-www-form-urlencoded"},
+        data={
+            "grant_type": "client_credentials",
+            "client_id": settings.INTEGRATION_TEST_CLIENT_ID,
+            "client_secret": settings.INTEGRATION_TEST_CLIENT_SECRET,
+        },
+    ).json()["access_token"]
+
+
+@pytest.fixture
+def command_type_map() -> dict[str, type]:
+    return {
+        "AutocompleteField": str,
+        "MultiLineTextField": str,
+        "TextField": str,
+        "ChoiceField": str,
+        "DateField": datetime,
+    }
+
+
+@pytest.mark.parametrize(
+    "Command,uuid",
+    [
+        (AssessCommand, "446812b3-8486-4e6c-946c-d971a1a247f7"),
+        (GoalCommand, "4dd6d032-0da5-4e7f-a834-161161c971d7"),
+        (HistoryOfPresentIllnessCommand, "d3f904ba-a425-4605-9030-2eb29673c89d"),
+        (MedicationStatementCommand, "1e41d72a-62fb-4d46-83d2-d79d030f54cb"),
+        (PlanCommand, "b381862c-4cbf-4ffc-9398-d8a0989a941f"),
+        (QuestionnaireCommand, "a6a58bc2-f095-43ee-9cda-edba0db861ac"),
+        (ReasonForVisitCommand, "9a9c7a79-816d-4618-a47f-a886a1acc1cf"),
+        (StopMedicationCommand, "e041315d-74c7-468d-bac9-b814013123d8"),
```

Comment:
How can we account for different UUIDs on different Canvas installations? Could we use the Commands HTTP API to create them for the test?

Reply:
that's a very good idea. i'll do that